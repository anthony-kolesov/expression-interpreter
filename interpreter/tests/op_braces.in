# Confirm that braces change priority of operations.
out 4 + 5 * 3
print "\n"

out (4 + 5) * 3
print "\n"

out 4 + (5 * 3)
print "\n"
