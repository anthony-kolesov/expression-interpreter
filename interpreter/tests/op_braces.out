19
27
19
